scripts/ci/lib.sh

```diff
@@ -470,7 +470,7 @@ mark_collector_release() {
     info "Create a branch for the PR"
 
     collector_version="$(cat COLLECTOR_VERSION)"
-    cd /tmp/collector || exit
+    pushd /tmp/collector || exit
```

Comment:
Secondary concern, not your change: `|| exit` seems wrong here because it will return silently with a _success_ exit code 0. I think this should be `|| die "Could not cd into tmp directory"` or `|| exit 1` or so.